docs/components/vxp-label.md

```diff
@@ -2,24 +2,25 @@
 sidebarDepth: 0
 ---
 
-# Label
+# VxpLabel
 ## Live Demo
 
 <DocExampleBox :liveDemoMode="true">
 ```html
-<Label text="Label" />
+<VxpLabel text="Label" />
 ```
-<LabelLiveDemo />
+<VxpLabelLiveDemo />
 </DocExampleBox>
 
-This is an overview of the most common usage of `<Label>`.
+This is an overview of the most common usage of `<VxpLabel>`.
 
-`Label` is a UI component that displays read-only text.
+`VxpLabel` is a UI component that displays read-only text.
 
 ## Props
 
 | Name     | Type    | Default | Web | Mobile |
 | -------- | ------- | ------- | --- | ------ |
 | text     | String  |         | ✔   | ✔      |
```

Comment:
can you please add description column, it has been probably forgotten

Reply:
description column is added